src/app/material-docs-app.ts

```diff
@@ -1,6 +1,6 @@
 import {Component, ViewEncapsulation} from '@angular/core';
 import {Router, NavigationEnd} from '@angular/router';
-import 'rxjs/add/operator/filter';
+import {filter} from 'rxjs/operators';
```

Comment:
This should be imported from `rxjs/operators/filter`, otherwise it'll import all the operators. Same goes for all the other `rxjs/operators` imports.

Reply:
Good call. Fixed.